BraveShared/Analytics/DAU.swift

```diff
@@ -199,47 +221,31 @@ public class DAU {
         if firstPing {
             return dauParams(true, true, true)
         }
-        
-        let month = todayComponents.month
-        let year = todayComponents.year
-        
+
         guard let stat = dauStat?.compactMap({ $0 }) else {
             log.error("Cannot cast dauStat to [Int]")
             return nil
         }
         
-        guard stat.count == 3 else {
-            log.error("dauStat array must contain exactly 3 elements")
+        guard let lastPingStat = stat.first else {
+            log.error("Can't get last ping timestamp from dauStats")
             return nil
         }
         
-        let _month = stat[1]
-        let _year = stat[2]
+        let lastPingDate = Date(timeIntervalSince1970: TimeInterval(lastPingStat))
         
-        let lastPingDate = Date(timeIntervalSince1970: Double(stat[0]))
+        let pings = getPings(forDate: today, lastPingDate: lastPingDate)
```

Comment:
As discussed, `today` should probably be either removed or recalculated as `Date()`

Reply:
We should put it here https://github.com/brave/brave-ios/pull/794, since this PR changes the DAU from multiple instances to one per app